packages/veritone-react-common/src/components/FaceEngineOutput/RecognizedFaceMatch/index.js

```diff
@@ -29,7 +29,7 @@ class RecognizedFaceMatch extends Component {
           <div className={styles.entityFullName}>{entity.fullName}</div>
         </div>
         <div>
-          {confidence && (
+          {!!confidence && (
```

Comment:
why !! ?

Reply:
Because is confidence is undefined then react will render it as a 0. I could use lodashs !isUndefined(confidence) to make it more clear.

Reply:
Thanks! I am a layperson to React so please do it in your way. :)  I saw `&&` behind it so thought confidence will be boolean for sure.